python/xorbits/_mars/dataframe/datasource/tests/test_datasource.py

```diff
@@ -651,3 +653,57 @@ def test_merge_small_files():
         df2.chunks[1].index_value.to_pandas(), pd.RangeIndex(8, 16)
     )
     assert df2.nsplits == ((8, 8), (4,))
+
+
+def test_read_parquet_estimate_size():
+    test_df = pd.DataFrame(
+        {
+            "a": np.arange(10).astype(np.int64, copy=False),
+            "b": [f"s{i}" for i in range(10)],
+            "c": np.random.rand(10),
+        }
+    )
+
+    with tempfile.TemporaryDirectory() as tempdir:
+        file_path = os.path.join(tempdir, "test.parquet")
+        test_df.to_parquet(file_path)
+
+        df = read_parquet(file_path)
+        tiled = tile(df)
+        sizes = dict()
+        chunk = tiled.chunks[0]
+        estimate_size(sizes, chunk.op)
+        estimated_size = sizes[chunk.key][0]
+        assert estimated_size > test_df.memory_usage(deep=True).sum()
+
+        df = read_parquet(file_path)
```

Comment:
Is this exactly the same test as line 671-677?

Reply:
Removed